tests/org.eclipse.e4.ui.tests.css.core/src/org/eclipse/e4/ui/tests/css/core/parser/MediaRulesTest.java

```diff
@@ -29,18 +30,23 @@
 public class MediaRulesTest {
 
 	@Test
-	@Disabled("//THIS TEST KNOWN TO FAIL Dec 16/08")
 	void testMediaRule() throws Exception {
 		String css = """
-			@media screen, print {
-			BODY { line-height: 1.2 }
-			}
-			Label { background-color: #FF0000 }""";
+				@media screen, print {
+					BODY { line-height: 1.2 }
+					Label { background-color: #FFFFFF }
+				}
+				BODY { line-height: 1.3 }
+				Label { background-color: #FF0000 }""";
 		CSSStyleSheet styleSheet = ParserTestUtil.parseCss(css);
 		assertNotNull(styleSheet);
-		CSSRuleList rules = styleSheet.getCssRules();
-
+		List<String> cssRules = IntStream.range(0, styleSheet.getCssRules().getLength())
+				.mapToObj(i -> styleSheet.getCssRules().item(i).getCssText()).toList();
 
-		assertEquals(1, rules.getLength());
+		assertEquals(1, cssRules.stream().filter(it -> it.contains("line-height")).toList().size());
+		assertEquals(1, cssRules.stream().filter(it -> it.contains("background-color"))
+				.toList().size());
+		assert (cssRules.contains("background-color: #FF0000"));
+		assert (cssRules.contains("line-height: 1.3"));
```

Comment:
This one is the wrong `assert`, you probably meant `assertTrue`. In any case, please use HarmCrest / AssertJ assertions, they read better.
```java
import static org.assertj.core.api.Assertions.assertThat;
...
      assertThat(cssRules).contains(...); // you could also use "containsOnly" or some other one
```